src/test/java/net/datafaker/providers/videogame/MarvelSnapTest.java

```diff
@@ -0,0 +1,32 @@
+package net.datafaker.providers.videogame;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MarvelSnapTest extends VideoGameFakerTest {
+
+    @Test
+    void charactersTest() {
+        String character = faker.marvelSnap().character();
+        assertThat(character).matches("[ A-Za-z']+");
+    }
+
+    @Test
+    void zonesTest() {
+        String zone = faker.marvelSnap().zones();
+        assertThat(zone).matches("[ A-Za-z]+");
```

Comment:
this regular expression is incorrect. I doesn't accommodate for Kamar-Taj. Can you please change the `@Test` annotations to `@RepeatedTest(10)`. That has a far higher chance of spotting these issues, and it's fast enough.